TF_Settings_Web/src/Pages/About/AboutScreen.tsx

```diff
@@ -0,0 +1,114 @@
+        }
+
+        const onServiceChangeHandler = TouchFree.RegisterEventCallback('OnServiceStatusChange', setVersionInfo);
+
+        return () => {
+            onServiceChangeHandler.UnregisterEventCallback();
+        };
+    }, []);
+
+    return (
+        <>
+            <TabBar />
+            <div className={classes('container')}>
+                <div className={classes('title-line')}>
+                    <h1> About </h1>
+                </div>
+                <div className={classes('info-table')}>
+                    <InfoTextEntry title="TouchFree Version" text={tFVersion} />
+                    <InfoTextEntry title="Tracking Version" text={trackingVersion} />
+                    <InfoTextEntry title="Camera Firmware Version" text={cameraFWVersion} />
+                    <InfoTextEntry title="Camera Serial Number" text={cameraSerial} />
+                </div>
+                <div className={classes('page-divider')} />
+                {ENABLE_ADVANCED_ABOUT && !isBrightSign && (
+                    <>
+                        <div className={classes('title-line')}>
+                            <h1> Advanced </h1>
+                        </div>
+                        <div className={classes('info-table')}>
+                            <InfoButtonEntry
+                                title="Tracking Log Files"
+                                buttonTitle="Show Tracking Log Files"
+                                onClick={() => {
+                                    console.log('C:/ProgramData/Ultraleap/HandTracker/Logs');
+                                }}
+                            />
+                            <InfoButtonEntry
+                                title="TouchFree Log Files"
+                                buttonTitle="Show TouchFree Log Files"
+                                onClick={() => console.log('C:/ProgramData/Ultraleap/TouchFree/Logs')}
+                            />
+                        </div>
+                    </>
+                )}
+            </div>
+        </>
+    );
+};
+
+interface InfoTextEntryProps {
+    title: string;
+    text: string;
+}
+
+const InfoTextEntry: React.FC<InfoTextEntryProps> = ({ title, text }) => (
+    <div className={classes('info-table__entry')}>
+        <span className={classes('info-table__entry__title')}>{title}:</span>
+        <span className={classes('info-table__entry__text')}>{text}</span>
+    </div>
+);
+
+interface InfoButtonEntryProps {
+    title: string;
+    buttonTitle: string;
+    onClick: () => void;
+}
+
+const InfoButtonEntry: React.FC<InfoButtonEntryProps> = ({ title, buttonTitle, onClick }) => (
+    <div className={classes('info-table__entry')}>
+        <span className={classes('info-table__entry__title')}>{title}:</span>
+        <span className={classes('info-table__entry__button')}>
+            <TextButton title={buttonTitle} onClick={onClick} />
+        </span>
+    </div>
```

Comment:
:+1: Love this! This file is really nicely structured!

Reply:
Thank you - thinking about the BEM class names helped to structure it